pkg/config/toml/defaults/Ethereum_Mainnet.toml

```diff
@@ -5,6 +5,9 @@ MinContractPayment = '0.1 link'
 OperatorFactoryAddress = '0x3E64Cd889482443324F91bFA9c84fE72A511f48A'
 NoNewFinalizedHeadsThreshold = '9m'
 
+[Transactions]
```

Comment:
Should we keep this config in dedicated structure to signal that it's only related to the EVM Chain Capability and not to the TXM in general?

Reply:
[EVM.Transactions] defines the config for TXM. `ConfirmationTimeout` is a config option for EVM Chain Capability. We should probably move it to [EVM.ChainCapability] to make it clearer.